code/game/machinery/computer/camera.dm

```diff
@@ -78,6 +78,25 @@
 	else
 		..()
 
+/obj/machinery/computer/security/telescreen/attackby(obj/item/I, mob/user, params)
+	if(ismultitool(I))
+		var/direction = input(user, "Which direction?", "Select direction!") as null|anything in list("North", "East", "South", "West", "Centre")
+		if(!direction)
+			return

```

Comment:
Not sure if it's a big deal, but someone could multitool a console, walk away a bit, and then fill in the prompt, to make it change position remotely

Reply:
Oh, good point. Will update